spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java

```diff
@@ -288,7 +288,7 @@ private SpoonDeviceRunner getTestRunner(String serial, SpoonInstrumentationInfo
     private String methodName;
     private boolean noAnimations;
     private IRemoteAndroidTestRunner.TestSize testSize;
-    private int adbTimeout = DEFAULT_ADB_TIMEOUT;
+    private int adbTimeout = DEFAULT_ADB_TIMEOUT_SEC * 1000;
```

Comment:
I'm fine with the method name staying the same. It has a whole slew of problems I'm not interested in fixing at this point.